components/style/values/computed/mod.rs

```diff
@@ -586,3 +589,45 @@ impl ClipRectOrAuto {
 
 /// <color> | auto
 pub type ColorOrAuto = Either<Color, Auto>;
+
+/// The computed value of a CSS `url()`, resolved relative to the stylesheet URL.
+#[cfg(feature = "servo")]
+#[derive(Clone, Debug, HeapSizeOf, Serialize, Deserialize, PartialEq)]
+pub enum ComputedUrl {
```

Comment:
Yeah, I meant this, but I wouldn't worry about that for this PR, it's fine to do it in followup work.